.github/workflows/test-e2e-runtime.yaml

```diff
@@ -0,0 +1,30 @@
+name: Func E2E Lifecycle Test
```

Comment:
Why "Lifecycle" in name here @jrangelramos ?

Reply:
Well.. it is because there is 2 sets of e2e tests: the lifecycle tests (e2e_lifecycle_tests.sh) and extended tests (e2e_extended_tests.sh)

The lifecycle one tests a function lifecycle such as, creation, deployment, read, update (change function and redeploy) and deletion.. the CRUD. against a specific language runtime and template.

The extended tests runs other sort of tests such as invoke, config labels, volumes, etc.

The workflow in this case will run the lifecycles tests for all runtimes.